docs/rest_api.md

```diff
@@ -608,6 +616,32 @@ Authorization requirements:
 
 * Administrative access.
 
+#### node_set_metadata
+
+`PUT /node/<node>/metadata/<label>`
+
+Request Body: 
+
+	{
+		"value": <value>
+	}	
```

Comment:
One potential issue - I don't think sure arg verification will work with arbitrary body args, since it wants to verify each key. This is a bummer, because otherwise the programmer will have to make an extra step to get rid of the `value:stuff` for the values.

One way to address this would be to add an optional parameter to `@rest_call` like `literal_body`, where the body doesn't get interpreted and gets passed using some fixed name.

Given the trickiness of that code, my thought is to keep this the way it is now, and open an issue to enable arbitrary HTTP bodies to be passed through in the future.

@zenhack - what do you think?